packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -759,6 +759,41 @@ export class SankeyChartBase extends React.Component<ISankeyChartProps, ISankeyC
         borderColorsForNodes,
       );
 
+      const nodeLinkDomOrderArray: { layer: number; type: string; index: number }[] = [];
+      nodes.sort((a: SNode, b: SNode) => {
```

Comment:
Can you add in description what is fixed and what is left.

Reply:
done